From eb5ffe2b4177eb631821f822f2b051a9255d9f24 Mon Sep 17 00:00:00 2001
From: Shengzhou Liu <Shengzhou.Liu@freescale.com>
Date: Fri, 10 Aug 2012 18:48:31 +0800
Subject: [PATCH 288/430] powerpc/p4080ds: dts - add usb controller version
 info and port0

commit 09a3017a585eb8567a7de15b426bb1dfb548bf0f upstream

Add the missing usb controller version info and port0, which is
required during setup usb phy.

Signed-off-by: Shengzhou Liu <Shengzhou.Liu@freescale.com>
Signed-off-by: Kumar Gala <galak@kernel.crashing.org>
Signed-off-by: Wei Yang <Wei.Yang@windriver.com>
---
 arch/powerpc/boot/dts/fsl/p4080si-post.dtsi |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/fsl/p4080si-post.dtsi b/arch/powerpc/boot/dts/fsl/p4080si-post.dtsi
index ba0f088..4331bd9 100644
--- a/arch/powerpc/boot/dts/fsl/p4080si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/p4080si-post.dtsi
@@ -354,10 +354,12 @@
 /include/ "qoriq-gpio-0.dtsi"
 /include/ "qoriq-usb2-mph-0.dtsi"
 	usb@210000 {
+		compatible = "fsl-usb2-mph-v1.6", "fsl,mpc85xx-usb2-mph", "fsl-usb2-mph";
 		port0;
 	};
 /include/ "qoriq-usb2-dr-0.dtsi"
 	usb@211000 {
+		compatible = "fsl-usb2-dr-v1.6", "fsl,mpc85xx-usb2-dr", "fsl-usb2-dr";
 	};
 /include/ "qoriq-sec4.0-0.dtsi"
 /include/ "qoriq-pme-0.dtsi"
-- 
1.7.5.4

